.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+
+node_js:
+  - 6
```

Comment:
Prepare to have your day made:
Travis-CI is indeed installing Node 6.1 (current stable branch), and npm 3.8.6:
```bash
$ node --version
v6.1.0
$ npm --version
3.8.6
$ nvm --version
0.31.0
```